rosetta-2021.html

```diff
@@ -0,0 +1,236 @@
+<script>
+  // Rule of thumb: put a semicolon at the end of every line that doesn't end in a curly
+
+  // Simple output
+  // =============
+
+  // puts "howdy!"
+
+  // alert("howdy!");
```

Comment:
Maybe `console.log()` would be less annoying than `alert()`?

Reply:
Yes, it would definitely be less annoying, but you need to you dev tools open to see the output in the browser— if that's where you're running the JS.